crypto-ffi/src/generic.rs

```diff
@@ -373,11 +373,13 @@ impl CoreCrypto<'_> {
                 self.central
                     .lock()
                     .map_err(|_| CryptoError::LockPoisonError)?
-                    .encrypt_message(conversation_id, message),
+                    .encrypt_message(&conversation_id, message),
             ),
         )
     }
 
+    #[deprecated(note = "Will be covered by decrypt_message return type")]
+    #[allow(deprecated)]
     pub fn conversation_exists(&self, conversation_id: ConversationId) -> bool {
```

Comment:
Why deprecate this method? It's useful for very simple & cheap checks (in cases for instance where the backend would mis-order the arrival of websocket events and welcome messages, to inform CoreLogic's decision making)

Reply:
Ok I thought about moving this to decrypt_message and return a boolean to inform CoreLogic when the client has been "self removed" from the group.
But ok to keep it.